### ISSUE TITLE
Add note that cobalt 0.17.5 has not been released, which will cause local build problems

### ISSUE THREAD

**Issue by quietlychris** [anowell/are-we-learning-yet#113]
As of writing (9 December 2021), [`cobalt`](https://cobalt-org.github.io/) `v0.17.5` is not publicly available through the Releases page, and the most recent `v0.17.4` throws an error when attempting to build. It took a few minutes to figure out why (I thought maybe the README page had a typo on the version number) before I looked in the CI config file to find that Cobalt was actually being installed from the Git repository using 
```sh
$ cargo install \
            --git https://github.com/cobalt-org/cobalt.rs \
            --rev 6047852d62aa505a4dc6a82715485d465be5c596
```

Until the next version is released (although Cobalt has a history of regular releases, I can not find a planned schedule), a note should be added to the README on how to install the required version.

**Comment by quietlychris** [comment]
(I will submit a PR accordingly, but wanted to put this out there in case anyone had this issue in the meantime)

**Comment by anowell** [comment]
This is correct. For context: the footer includes a "last updated" date that maps to the date that the site was last regenerated, and cobalt [coincidentally] added support for that in the past week via https://github.com/cobalt-org/cobalt.rs/pull/941

While not urgent, I'll comment over there that we're already depending on it and would love a new cobalt release with this change.

**Comment by quietlychris** [comment]
Addressed in #114
